mood-analysis.rb

```diff
@@ -28,5 +65,14 @@ def analyze_mood(words)
   "05/11 Yay, yay, yay! I'm having a awfuly great day."
 ]
 
-puts analyze_mood(text[0])
-puts analyze_mood(text[1])
+result = text.map do |day|
+  date = day.split(" ").first
+  mood = analyze_mood(day)
+  puts
+  puts "#{date} #{mood}"
+
+  [date, mood]
+end
+
+puts "It takes #{happy_days(result)} entries for 3 happy days to occur"
+puts "The most common mood is #{overall_mood(result)}"
```

Comment:
Yes! Good job returning the smallest piece of information necessary and not an entire string encapsulating that information!